.changeset/yellow-kings-rhyme.md

```diff
@@ -0,0 +1,6 @@
+---
+'@astrojs/webapi': patch
+'@astrojs/telemetry': patch
```

Comment:
In the previous `undici` bump, I forgot to add a changeset for `@astrojs/telemetry`, so it didn't update for users